components/delegations/DelegateCard.tsx

```diff
@@ -66,7 +67,12 @@ export default function DelegateCard({ delegate }: PropTypes): React.ReactElemen
 
           <Box sx={{ mt: 3 }}>
             {showLinkToDetail && (
-              <Link href={`/delegates/${delegate.voteDelegateAddress}`}>
+              <Link
+                href={{
+                  pathname: `/delegates/${network}/${delegate.voteDelegateAddress}`,
+                  query: { network }
```

Comment:
delegates/network/ path was introduced to allow server side rendering with static properties

Reply:
I'm curious to hear what others think but having network in the path feels clunky and somewhat unnecessary to me. As the devs here we use kovan a fair amount but I think 99% of our users will be on mainnet. We shouldn't optimize for using testnets. I think we should keep the network out of the path name and just use the query param. 

If we have to refetch data client side because we're on a testnet, I don't think it's a big deal. We are some of the only users who are using kovan in this case.

Reply:
Yeah I would prefer not to have the network in the path. Changes to the URL might be something we would want to run by governance also.